docs/tutorials/configure/scuttlebot.md

```diff
@@ -258,7 +258,7 @@ If you hear a "whining" sound from the motor, the power level is not high enough
 If this happens, increase the power level by 10% increments until it starts to turn.
 {{< /alert >}}
 
-If your wheel turns in reverse when it should turn forward, add the `dir_flip` attribute to the motor's configuration, by clicking **Show more** and setting the attribute to "true."
+If you want your wheel to turn in reverse, you can add the `dir_flip` attribute to the motor's configuration, by clicking **Show more** and setting the attribute to "true."
```

Comment:
I think it makes sense to call this out as something people will encounter as opposite to what they're expercting
```suggestion
If you wheel turns in the opposite in reverse when it should turn forward, you can add the `dir_flip` attribute to the motor's configuration, by clicking **Show more** and setting the attribute to "true."
```

Reply:
i was just not sure since we give them the exact a and b inputs, it wouldn't turn reverse?

Reply:
I think @JessamyT said it is possible to flip these accidentally? I think? Possible I am wrong. Let's wait for Jessamy to reply

Reply:
@JessamyT

Reply:
I think that since DC motors don’t have an inherent “forward”, they might be flipped?? I think this sentence shouldn’t change from how it was before, because people wouldn’t want it to turn in reverse—it’s just if it is turning in reverse and they need to fix it.

Reply:
As mentioned above, my SCUTTLE is configured in the way this page said before this PR's changes (with A 16, B 15), no dir flip, and it drives forwards properly. @mestcihazal do you have SCUTTLEs that have 15, 16 and no dir flip, and drive forward? At one point I know some of the NYC office scuttles that people were learning on were driving backwards (with the big wheels in the back) and had the cameras by the caster wheels.

@npentrel your suggestion seems to have a typo but what do you think of the original wording?

Reply:
the one I just configured has 15, 16, and no dir flip, and it drives forward (same with old tipsy)

Reply:
Just looked in the SCUTTLE wiring guide. When they talk about motor driver power wires, they say "Apply +5V to the red wire and select terminal which gives CCW rotation." So it's kinda up to the user to assemble it one way or the other, and they do specify, but people could totally do it the other way on accident. Also mine came pre-assembled by scuttle the way that it is--maybe yours did too.

Reply:
original wording is fine by me

Reply:
```suggestion
If your wheel turns in reverse when it should turn forward, add the `dir_flip` attribute to the motor's configuration, by clicking **Show more** and setting the attribute to "true."
```